extensions/frontend-tiles/package.json

```diff
@@ -46,6 +48,8 @@
     "@itwin/core-common": "workspace:*",
     "@itwin/core-frontend": "workspace:*",
     "@itwin/core-geometry": "workspace:*",
+    "@itwin/core-webpack-tools": "workspace:*",
```

Comment:
we dont seem to be using this dep here anymore?

Reply:
removed in latest commit